components/layouts/services/services.hbs

```diff
@@ -5,13 +5,14 @@
     </hgroup>
 
     <article class="service" data-id="services-1">
-      <div class="service__text-content">
+      <a class="service__text-content" href="#">
         <div class="service__enumerator">01 / 03</div>
-        <h4 class="service__heading"><a href="#">Marketing, Creative, & XD
-            Talent</a></h4>
+        <div class="service__heading__wrapper">
+          <span class="service__heading">Marketing, Creative, & XD Talent</span>
+        </div>
         <p class="service__text">Lorem ipsum, dolor sit amet consectetur
           adipisicing elit. Quo iusto error explicabo!</p>
-      </div>
+      </a>
```

Comment:
This change is causing a bug that will have to be addressed. There's a underline caused by the `div` changed to an `a`. 
![image](https://user-images.githubusercontent.com/5826750/146954215-12ded07a-20c4-42ca-af8b-4b054095f6ac.png)

Reply:
oh my bad I haven't updated the `hbs` file. On the pattern vtl file I have the `.service` an anchor tag instead of the `service__text-content`. Thanks for the fix Nick